examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>We miss another headline - so far we have only "Downloading the datase" and everything is in that headline.</p><p> </p><p>I think we need to add a new one here: Training a TensorFlow Keras Model with Merlin dataloader</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>good point, will pay more attentiont to those headlines going forward</p>